prod/westeurope/internal/api/apim/api_management_api_io_payportal/policy.xml

```diff
@@ -0,0 +1,31 @@
+<policies>
+  <inbound>
+    <base />
+    <set-backend-service base-url="{{io-fn-pay-portal-key-url}}/api/v1" />
+    <set-header name="x-functions-key" exists-action="override">
+      <value>{{io-fn-pay-portal-key-key}}</value>
+    </set-header>
+    <rate-limit-by-key calls="20" renewal-period="30" counter-key="@(context.Request.IpAddress)" />
+    <cors>
+      <allowed-origins>
+        <origin>https://io-p-cdnendpoint-iopayportal.azureedge.net/</origin> <!-- CDN frontend io-payportal -->
+      </allowed-origins>
+      <allowed-methods>
+        <method>POST</method>
+        <method>OPTIONS</method>
+        <method>GET</method>
+      </allowed-methods>
+      <allowed-headers>
+        <header>contentType</header>
+        <header>Authorization</header>
```

Comment:
Are these http headers really required ? I am not sure about it.

Reply:
It isn't required. I'll remove it